cartography/intel/semgrep/__init__.py

```diff
@@ -20,4 +23,13 @@ def start_semgrep_ingestion(
     if not config.semgrep_app_token:
         logger.info('Semgrep import is not configured - skipping this module. See docs to configure.')
         return
-    sync(neo4j_session, config.semgrep_app_token, config.update_tag, common_job_parameters)
+
+    # fetch and load the Semgrep deployment
+    semgrep_deployment = get_deployment(config.semgrep_app_token)
+    deployment_id = semgrep_deployment["id"]
+    deployment_slug = semgrep_deployment["slug"]
+    load_semgrep_deployment(neo4j_session, semgrep_deployment, config.update_tag)
+    common_job_parameters["DEPLOYMENT_ID"] = deployment_id
+
+    sync_dependencies(neo4j_session, config.semgrep_app_token, config.update_tag, common_job_parameters)
+    sync_findings(neo4j_session, config.semgrep_app_token, config.update_tag, common_job_parameters, deployment_slug)
```

Comment:
It's awkward to need to pass `deployment_slug` as a parameter, should I add the slug to `common_job_parameters` instead? I didn't take that approach since I wasn't sure if it would have other side effects

Reply:
Yeah, put it in common_job_parameters. We do that with AWS_ID too